openfpga/src/fpga_verilog/verilog_top_testbench.cpp

```diff
@@ -1431,7 +1431,13 @@ void print_verilog_full_testbench_configuration_chain_bitstream(std::fstream& fp
   fp << ";" << std::endl;
 
   fp << "\t";
-  fp << "end else begin";
+  fp << "end else if (";
```

Comment:
It is better to leave a code comments saying that the 

``else if`` condition is required by Modelsim to synthesis the Verilog correctly.

Reply:
you mean it as a commit comment or actual verilog comment?

Reply:
Just code comment in the .cpp file